### PR TITLE
Catalog instances in plan context

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Instances tab is working in specific plan context
- Add variants list to breadcrumb

**Related issue(s)**

### CODE REVIEW

File: service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.js [hunk 1]
```diff
@@ -19,22 +19,35 @@ function getTabElementsIndicator(instancesCount) {
...
+
+const ServiceClassTabs = ({ serviceClass, currentPlan }) => {
   const assetGroup =
     serviceClass && (serviceClass.assetGroup || serviceClass.clusterAssetGroup);
 
-  const additionalTabs = serviceClass.instances.length
+  const instances = currentPlan
+    ? filterServiceInstances(serviceClass.instances, currentPlan)
+    : serviceClass.instances;
+  const additionalTabs = instances.length
```
Comment (parostatkiem) [inline service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.js]: I believe a separate variable is not necessary in this case.

File: service-catalog-ui/catalog/src/components/ServiceClassDetails/test/ServiceClassDetails.test.js [hunk 2]
```diff
@@ -98,8 +98,8 @@ describe('Service Class Details UI', () => {
       });
     });
 
-    it('Shows API package icon when the label is present', async () => {
-      const { queryByLabelText } = render(
+    it('Shows API package icon and breadcrumb when the label is present', async () => {
+      const { queryByLabelText, queryByText, debug } = render(
```
Comment (parostatkiem) [inline service-catalog-ui/catalog/src/components/ServiceClassDetails/test/ServiceClassDetails.test.js]: Is `debug` needed?

File: service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassDetailsHeader/ServiceClassDetailsHeader.component.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import LuigiClient from '@kyma-project/luigi-client';
-import { Breadcrumb } from '@kyma-project/react-components';
-
+import { Breadcrumb } from 'fundamental-react';
```
Comment (parostatkiem) [inline service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassDetailsHeader/ServiceClassDetailsHeader.component.js]: 🎉

File: service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.js (hunk 1 above)
Comment (parostatkiem) [reply service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.js]: But `instances.length` isn't a big conditional variable, is it?

File: service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassDetails.js [hunk 4]
```diff
@@ -68,6 +68,7 @@ export default function ServiceClassDetails({ name, plan }) {
   const serviceClassDisplayName = getResourceDisplayName(serviceClass);
 
   const serviceClassDescription = getDescription(serviceClass);
+  const serviceClassName = serviceClass.name;
```
[the remark this answers is not in the document]
Comment (akucharska) [reply service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassDetails.js]: This way it passes value and not reference

[the remark this answers is not in the document]
Comment (parostatkiem) [reply service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassDetails.js]: It'd be passed as a value anyway. JS passes function arguments by value if their type is non-reference (this case).

File: service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.js (hunk 1 above)
Comment (parostatkiem) [reply service-catalog-ui/catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.js]: Oh, sorry, I didn't notice the bottom part 🙈

**Pull request merged**